server/app.js

```diff
@@ -1,9 +1,21 @@
+const path = require('path');
 const express = require('express');
+
 const app = express();
 
 // Middleware
 app.use(express.static('dist'));
 app.use(express.json());
 app.use(express.urlencoded({ extended: false }));
 
+// Pls Put all other HTTP request above this GET request &&
+// Pls use endpoints/routes that is different than the Frond End React Router routes
```

Comment:
Lines 11 and 12 not needed. Server will be on separate instance as separation of concerns and routes will not interfere with React Router